backstage-helm/values.yaml

```diff
@@ -49,9 +49,10 @@ upstream:
               rejectUnauthorized: true
 
       integrations:
-        github:
-          - host: github.com
-            token: ${GITHUB_TOKEN}
+        bitbucketServer:
+          - host: {{.Values.global.clusterRouterBase }}
```

Comment:
I don't think this is correct.  If you look at David's route it is something like this:
https://bitbucket-bitbucket.apps.cluster-kbrvx.dynamic.opentlc.com/

So I believe you may need to prefix it with `bitbucket-bitbucket`.  So something like:
```
- host: bitbucket-bitbucket{{.Values.global.clusterRouterBase }}
```
same for the `apiBaseUrl` below

Reply:
done